### PR TITLE
document /version endpoint

### PR DESCRIPTION
**Pull request by go-oleg**
@mstemm

### CODE REVIEW

File: docs/jobs-api.md [hunk 1]
```diff
@@ -285,3 +285,20 @@ GET /api/v0/paths/forever.juttle HTTP/1.1
     }
 }
 ```
+
+###GET /version
```
Comment (demmer) [inline docs/jobs-api.md]: Should have noticed this in the original PR adding the route, but why not put this under `/api/v0/version`? 

Also I kinda think this should be named `/api/v0/version-info`.

Comment (go-oleg) [inline docs/jobs-api.md]: I somehow thought that would make it look like version info of the `/api` itself as opposed to components the server is running on. Don't have a strong preference though. Will change.

**Comment by go-oleg** [comment]
updated to document endpoint at `/api/v0/version-info` as implemented in https://github.com/juttle/juttle-service/pull/62

**Comment by mstemm** [comment]
lgtm.

**Pull request merged**